src/VisualStudio/CSharp/Impl/ProjectSystemShim/EntryPointFinder.cs

```diff
@@ -19,7 +19,7 @@ public static IEnumerable<INamedTypeSymbol> FindEntryPoints(INamespaceSymbol sym
         {
             var visitor = new EntryPointFinder();
```

Comment:
I don't get why a `FindEntryPoints` method needs an `INamespaceSymbol`? Shouldn't a `Compilation` be passed instead?

Reply:
So, even though this is an internal API this is actually a public contract with the project system. If we update this we need to have a co-ordinated insertion with the proejct system of VS will be broken. I'll file a bug to track updating this but I want this bugfix to not be blocked but the API change